packages/schematics/angular/migrations/update-6/index.ts

```diff
@@ -724,6 +724,43 @@ function updateTsLintConfig(): Rule {
   };
 }
 
+function updateRootTsConfig(): Rule {
+  return (host: Tree, context: SchematicContext) => {
+    const tsConfigPath = '/tsconfig.json';
+    const buffer = host.read(tsConfigPath);
+    if (!buffer) {
+      return;
+    }
+
+    const tsCfgAst = parseJsonAst(buffer.toString(), JsonParseMode.Loose);
+    if (tsCfgAst.kind != 'object') {
+      throw new SchematicsException('Invalid tsconfig. Was expecting an object');
+    }
+
+    const compilerOptionsAstNode = findPropertyInAstObject(tsCfgAst, 'compilerOptions');
+    if (!compilerOptionsAstNode || compilerOptionsAstNode.kind != 'object') {
+      throw new SchematicsException(
+        'Invalid tsconfig "compilerOptions" property; expected an object.',
+      );
+    }
+
+    if (
+      findPropertyInAstObject(compilerOptionsAstNode, 'baseUrl') &&
+      findPropertyInAstObject(compilerOptionsAstNode, 'module')
+    ) {
+      return host;
+    }
+
+    const compilerOptions = compilerOptionsAstNode.value;
+    compilerOptions.module = compilerOptions.module || 'es2015';
+    compilerOptions.baseUrl = compilerOptions.baseUrl || './';
+
+    host.overwrite(tsConfigPath, JSON.stringify(tsCfgAst.value, null, 2));
```

Comment:
I tired to use the JSON AST for this, but updating multiple properties within the same JSON Object is kinda tricky with `appendPropertyInAstObject` as it kept appending `,` after the last existing property. I also tried to use the `.remove` and `insertLeft`, but it kinda get too much complicated for this trivial thing.